tests/testenv/cli_test_env.go

```diff
@@ -287,6 +289,10 @@ func (e *CLITest) Run(t *testing.T, expectedError bool, args ...string) (stdout,
 		c.Stderr = os.Stderr
 	}
 
+	if e.PassthroughStdin {
```

Comment:
instead of that, you can make PasshtroughStdin into a reader and do something like:

```
c.Stdin = e.NextCommandStdin
e.NextCommandStdin = nil
```